sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy

```diff
@@ -2071,6 +2109,40 @@ class BlockBlobAPITest extends APISpec {
         destinationProperties.getAccessTier() == AccessTier.COOL
     }
 
+    def "Upload from URL source oauth"() {
```

Comment:
consider moving source oauth tests to separate test class.
please annotate tests (or test class if it comes to that) with `@RequiredServiceVersion`.